flyrs/src/gen/pb_rust/flyteidl/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
```

Comment:
Can we use buf to generate the flyteidl code? Something like https://github.com/flyteorg/flyte/pull/5187

Reply:
Thanks @eapolinario , I think this can help! I've already [tried](https://buf.build/community/neoeinstein-tonic?version=v0.4.0) but I'm not able to generate Rust code correctly in my local machine.
Since you can compile, there shouldn't be many problems in CI.